### ISSUE TITLE
guests form

### ISSUE THREAD

**Issue by jaeming** [jaeming/RSVP#3]
forgot to add an attribute: email.
Also, what other attributes are needed? 
So far we have name and comment.

**Comment by jaeming** [comment]
Meal preferences?

**Comment by salexzee** [comment]
Yes. I have a friend who is getting married and that seems to be something important.

**Comment by jaeming** [comment]
Added email attribute!

**Comment by jaeming** [comment]
One thought, in the future fo v2, it'd be good if a event user could specify what form fields they needed or could add extra ones. Basically like a form builder. I sort of can't imagine how to implement that right now but could be fun to explore.

**Comment by jaeming** [comment]
Will add optional fields for 'number in your party', 'meal preferences', and 'other special requests'.  The event user will label them while creating the event (eg. 'chicken or fish') and then will show it and the optional input for it if it exists for the guest form...  Something like <% unless @event.meal_preferences.blank? %>

Later I definitely want to explore the concept of form building or customizing the form to a further extent but this is bound to be a bit of a time suck so will leave that for v2.  If this turns into a real service, it could possibly be a paid feature?
